ext/mysql2/result.c

```diff
@@ -2,7 +2,6 @@
 
 #include "mysql_enc_to_ruby.h"
 #define MYSQL2_CHARSETNR_SIZE (sizeof(mysql2_mysql_enc_to_rb)/sizeof(mysql2_mysql_enc_to_rb[0]))
-
```

Comment:
:scissors: ?

Reply:
Yes, please avoid unrelated whitespace changes outside the area of code being modified.